### PR TITLE
Remove merge cell.

### LINKED ISSUE #1216: Merge cell icon unintuitive, action not always apparent.
Showing the notebook off, we realized that the up arrow isn't really obvious for what it is doing, which is to merge the cell below into the cell above. I am opening this issue to discuss how to go about fixing the cell creator button to make it more first time user friendly.

### PR DESCRIPTION
**Pull request by jdetle**
- [x] I have read the [Contributor Guide](https://github.com/nteract/nteract/blob/master/CONTRIBUTING.md)

Closes #1216

### CODE REVIEW

**Comment by jdetle** [comment]
If we want to keep the actions/reducers/menu commands I can go in and redo this PR.

**Review by rgbkrk — APPROVED** [review]

**Comment by rgbkrk** [comment]
Actually, can you keep the down arrow in the octicons? We don't need to delete the icon.

File: packages/octicons/src/index.js [hunk 1]
```diff
@@ -55,13 +55,6 @@ export const CodeOcticon = (props: any) => (
   </SVGWrapper>
 );
 
-export const DownArrowOcticon = (props: any) => (
-  <SVGWrapper width={10} height={16} viewBox="0 0 10 16" outerProps={props}>
-    <title>Merge Cells</title>
-    <path fillRule="evenodd" d="M5 3L0 9h3v4h4V9h3z" />
-  </SVGWrapper>
-);
```
Comment (rgbkrk) [inline packages/octicons/src/index.js]: This icon can be kept for now, so we don't have to make a major version change.

Comment (jdetle) [reply packages/octicons/src/index.js]: Done

**Review by rgbkrk — APPROVED** [review]

**Comment by rgbkrk** [comment]
Thank you!

**Pull request merged**